ccd-definition/CaseEventToFields.json

```diff
@@ -374,5 +374,126 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseEventID": "uploadCourtBundle",
```

Comment:
Sounds silly to have event called 'Upload a court bundle' which has 4 additional documents. It will make case history look concussing. IMO we should only have one field upload in the event.

Also it is inconsistent because there are no additional documents. Please talk to UX/content team. We should advise them against that.

Reply:
It has to have the availability to upload all documents again in the event.

Reply:
Who said that? I am questioning fundamental requirement.

Reply:
Morning damian. Jason and Moray both said this and said that they want to be able to upload documents after submission.